crates/rpc-proto/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "miden-rpc-proto"
+version = "0.3.0"
+description = "Miden RPC message definitions"
+readme = "README.md"
+keywords = ["miden", "node", "protobuf", "rpc"]
```

Comment:
In addition to `keywords`, let's add `categories` field - it could look something like this:
```toml
categories = ["no-std"]
```
(not sure if any other categories would be applicable)

Reply:
Done